ifStatement.js

```diff
@@ -13,6 +13,10 @@ Note:
    You can change the value of the age variable to test different cases. 
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let userAge=20
+let minDrivingAge=18
+if(userAge>minDrivingAge){console.log("the person is old enough to drive");}
```

Comment:
Well done! Please pay attention to your code organization. When you write the block of code for the if statement, it's better to add it in the next line, not right next to the curly brackets. The same goes for the closing bracket. Like this:
if(userAge>minDrivingAge){
console.log("the person is old enough to drive");
}